private-templates-service/client/src/components/Common/AdaptiveCard/AdaptiveCard.tsx

```diff
@@ -73,15 +40,29 @@ export function renderAdaptiveCard(): any {
   }
 }
 
+interface Props {
+  toggleModal: () => void;
+  cardtemplate: any,
+}
+
+function processTemplate(temp: any): any {
+  const templateString = JSON.stringify(temp.json);
+  const replaceChar = templateString.replace(/\\\\\\/g, '');
+  const trimTemp = replaceChar.slice(3, replaceChar.length - 3);
+  const jsonTemp = JSON.parse(trimTemp);
+  const template = renderAdaptiveCard(jsonTemp);
+  return template;
+}
```

Comment:
Can you add a comment above this function explaining what it does and why we need it?

Reply:
Done.